index.html

```diff
@@ -48,8 +50,166 @@ <h3 class="heading-3">Let's connect</h3>
+    <p class="card-text">Back End Dev</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+
+    <div class="work-section">
+    <img class="cards" src="./images/snap-card-2.png" alt="avocode">
+    <h2 class="heading-2">Multi-Post<br>Stories</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img  class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang">html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+</div>
+  <!--- ===== About section ======= --->
+  <div class="about">
+    <h1 class="heading">About<br>Myself</h1>
+    <p class="paragraph">
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <h3 class="heading-3">Let's connect</h3>
+    <div class="icons">
+      <a href="#">
+        <img class="social-icons" src="./images/github-img.png" alt="github">
+      </a>
+      <a href="#">
+        <img class="social-icons" src="./images/linkedin-img.png" alt="Linkedin">
+      </a>
+      <a href="#">
+        <img class="social-icons" src="./images/fingure-img.png" alt="fingure icon">
+      </a>
+      <a href="#">
+        <img class="social-icons" src="./images/tweet-img.png" alt="twitter">
+      </a>
+      <a href="#">
+        <img class="social-icons" src="./images/medium-img.png" alt="medium icon">
+      </a>
+      </div>
+
+      <div id="resume">
+  <a href="#">
+    <button class="btn-resume">Get my Resume</button>
+  </a>      
+      </div>
+      <div class="arrows" id="first-arrow">
+       <div id="languages">
+        <h2 id="heading-2">Languages</h2>
+        <i class="fa fa-angle-down"></i>
+        </div>
+      </div>
+        <ul class="language">
+          <li class="languages-content">
+            <img src="./images/js.png">
+            <p>JavaScript</p>
+          </li>
+          <li class="languages-content">
+            <img src="./images/html.png">
+            <p>HTML</p>
+          </li>
+          <li class="languages-content">
+            <img src="./images/css.png">
+            <p>CSS</p>
+          </li>
+        </ul>
```

Comment:
- [ ] Kindly consider structuring your skills following way in order to follow best practices and semantics.
```
<ul>
  <li>Languages
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```